Geometry/HcalTowerAlgo/src/HcalDDDGeometryLoader.cc

```diff
@@ -145,7 +142,7 @@ void HcalDDDGeometryLoader::fill(HcalSubdetector          subdet,
 
 void
 HcalDDDGeometryLoader::makeCell( const HcalDetId& detId,
-				 const HcalCellType& hcalCell,
+				 HcalCellType hcalCell,
```

Comment:
@bsunanda - you are passing HcalCellType by value. const reference is more appropriate here.